website/blog/2023-05-01-package-exports-support.md

```diff
@@ -0,0 +1,229 @@
+---
+title: 'Package Exports Support in React Native'
+authors: [huntie]
+tags: [announcement, metro]
+date: 2023-05-01
+---
+
+# Package Exports Support in React Native
+
+With the release of React Native 0.72, Metro — our JavaScript build tool — now includes experimental support for the `package.json` [`"exports"` field](https://nodejs.org/docs/latest-v18.x/api/packages.html#exports). [When enabled](#enabling-package-exports-experimental), it adds the following functionality:
```

Comment:
```suggestion
With the release of React Native 0.72, Metro — our JavaScript build tool — now includes experimental support for the `package.json` [`"exports"` field](https://nodejs.org/docs/latest-v18.x/api/packages.html#exports). [When enabled](/blog/2023/05/01/package-exports-support#enabling-package-exports-experimental), it adds the following functionality:
```

This link also need to use full URL, since it will be before the truncate tag.